fusor-ember-cli/app/components/deployment-role.js

```diff
@@ -70,6 +70,11 @@ export default Ember.Component.extend({
     return avail;
   }),
 
+  // NOTE .on('didInsertElement')
+  updateRoleCountonDeployment: Ember.on('didInsertElement', function() {
+    return this.send('setRoleCountOnController', this.get('role').get('roleType'), this.get('roleNodeCount'));
```

Comment:
this action `setRoleCountOnController` is called each time the component is inserted into the DOM `on('didInsertElement')`

The action is called on the parent component `node-profile` which is then called on the `assigned-nodes` controller.